sensors/bms/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>bms</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="karate.martynas1@gmail.com">vortex</maintainer>
```

Comment:
lol yes, it is I that has written all the code, I take all the credit for the beutifull code hehehehe >:)

Reply:
teehee